rust/rope/src/engine.rs

```diff
@@ -222,6 +234,18 @@ impl Engine {
         self.rev_id_counter += 1;
     }
 
+    pub fn compare(&self, base_rev: usize, other_rev: usize) -> bool {
```

Comment:
nit: the name of this function doesn't convey what it _does_, e.g. that it isn't just comparing but is checking equality, and that it returns a bool. 

I'm not sure what I'd call it, but maybe something like `are_equal` or `are_equivalent`?